tests/AutoReview/FixerFactoryTest.php

```diff
@@ -264,13 +262,10 @@ public function testFixersPriorityPairsHaveIntegrationTest(FixerInterface $first
             'class_attributes_separation,braces.test',
             'class_attributes_separation,indentation_type.test',
             'indentation_type,phpdoc_indent.test',
-            'line_ending,single_blank_line_at_eof.test',
```

Comment:
there is prio issue (anymore) as both fixers use the same line break configuration